debian_package/sbin/folder2ram

```diff
@@ -234,67 +235,40 @@ exit
 EOF
 
   #chmodding this script to make it only root/group-writable and root/group-executable
-  chmod 774 "/etc/init.d/folder2ram_temporary"
+  chmod 755 "/etc/init.d/folder2ram_temporary"
 
 } #### END write_cleanup_initscript
 
 write_systemd_startup_service() {
 
-  if [ "$timeout_setting" != '' ]; then
-    timeout_line="TimeoutSec=$timeout_setting"
-  fi
+  #read timeout from configuration file
+  local timeOUT=$(read_config_file | grep "^TIMEOUT" | awk -F'=' '{print $2}')
+  [ "${timeOUT}" != "" ] || timeOUT=120
 
   #writing a tiny amount of stuff to have systemd do the same thing as above.
   #clear symptom that systemd is master race.
   cat <<EOF >"$systemd_startup_service_file"
 [Unit]
 Description=folder2ram systemd service
 After=local-fs.target
-After=blk-availability.service
```

Comment:
This was added to avoid breaking thins in OpenMediaVault and other distros, see commit here https://github.com/bobafetthotmail/folder2ram/commit/2dc9d6db65b77806350a0d1466ea382cab19f95d
and in OpenMediaVault plugins https://github.com/bobafetthotmail/folder2ram/commit/2dc9d6db65b77806350a0d1466ea382cab19f95d
Please don't delete this line